src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -33,20 +33,45 @@ public PythonCodeMigrationEventArgs(string oldCode, string newCode)
 
     public abstract class PythonNodeBase : VariableInputNode
     {
-        private PythonEngineVersion engine = PythonEngineVersion.Unspecified;
+        private string engine = string.Empty;
 
         [JsonConverter(typeof(StringEnumConverter))]
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Populate)]
-        [DefaultValue(nameof(PythonEngineVersion.IronPython2))]
+        [DefaultValue("IronPython2")]
```

Comment:
this is because ironPython project is not built?

Reply:
Because I deprecated PythonEngineVersion and I did not want to use it anymore
For some reason ..using the PackageEngineManager.IronPythonEngineName does not work...compiler complains that it is not a const expression (even though it is a static readonly string...)

Reply:
We decided to leave IronPython as the DefaultSerialized value...so that older graphs (that did not have an explicit engine serialized) can show the migration warnings